### PR TITLE
Part of #17670 - Replace Typography with Text component in file signature-request-data.js

### PR DESCRIPTION
**Pull request by PrgrmrHarshShukla**
## Explanation
This PR is a Part of #17670 Replace Typography with Text component

In this PR I have worked on the file
- **ui\components\app\signature-request\signature-request-data\signature-request-data.js**

### Before
Screenshot (22)

### After
Screenshot (23)

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Review by georgewrmarshall — COMMENTED** [review]
Hey @PrgrmrHarshShukla, good start! Looks like you are missing the `Text` import and you have some linting issues. To debug these you can use `yarn lint`

**Comment by PrgrmrHarshShukla** [comment]
Hey @georgewrmarshall ,
I have made the required changes.
Kindly review it.

**Review by georgewrmarshall — COMMENTED** [review]
Looks good just need to fix lint issue

**Comment by PrgrmrHarshShukla** [comment]
I have worked on the lint issue.

@georgewrmarshall 
Kindly review the changes.

**Review by georgewrmarshall — DISMISSED** [review]
LGTM! Thanks for your contribution 👍

**Comment by georgewrmarshall** [comment]
Just need to update snapshot `yarn jest ui/components/app/signature-request/signature-request-data --updateSnapshot`

**Comment by PrgrmrHarshShukla** [comment]
Finally I have been able to add the before and after screenshots also by running the storybook locally.

@georgewrmarshall 
Please check.

**Review by pedronfigueiredo — DISMISSED** [review]

File: ui/components/app/signature-request/signature-request-data/signature-request-data.js [hunk 1]
```diff
@@ -44,11 +44,11 @@ function SignatureRequestData({ data }) {
             }
```
Comment (DDDDDanica) [inline ui/components/app/signature-request/signature-request-data/signature-request-data.js:44]: we can also use new enum `FontWeight` here

**Review by georgewrmarshall — COMMENTED** [review]
Code looks great just need to update snapshots
```
yarn jest ui/components/app/signature-request/signature-request-data/signature-request-data.js --updateSnapshot
```

**Review by georgewrmarshall — APPROVED** [review]

**Review by NidhiKJha — APPROVED** [review]

**Review by NidhiKJha — APPROVED** [review]
LGTM! Need to wait for the builds to pass to merge this PR

**Comment by georgewrmarshall** [comment]
Looks like you may have a legitimate e2e failure because it's to do with a signature request component but I can't figure out what's causing it

Screenshot 2023-05-09 at 5 29 10 PM

**Comment by georgewrmarshall** [comment]
Hey @PeterYinusa, do you have some insight as to why the e2e tests are failing on this PR?

**Comment by PeterYinusa** [comment]
> Hey @PeterYinusa, do you have some insight as to why the e2e tests are failing on this PR?

From a quick glance, it looks like the test is looking for an element with a data attribute `[data-testid="signature-request-scroll-button"]`., but it does not appear after 10 seconds.

Were there any changes to the scroll button on the signature request screen? The test previously used to click this button before pressing the sign button

**Comment by georgewrmarshall** [comment]
Hey @PrgrmrHarshShukla, just wanted to check in. It looks like this PR is almost there. Did you still want to work on it? If so, that's great! Let us know if you need any assistance or if there are any blockers. If you're unable to continue working on it or haven't had a chance to address it, no worries! I can take it from here and carry it forward. Cheers!

**Comment by georgewrmarshall** [comment]
I think this legitimately has some e2e issue would you be able to investigate @PrgrmrHarshShukla?

**Comment by georgewrmarshall** [comment]
Hey @PrgrmrHarshShukla, thanks for all your effort with this PR I am going to close it as it has become stale. Please feel free to open another PR in future. Thanks for your contributions thus far!

**Pull request closed without merging**